### PR TITLE
Fix #10 - Remove z-index to stop textbox from drawing overtop other c…

### LINKED ISSUE #10: Relay address textboxes draw overtop other content.
On some pages with in-content popups, textboxes that are detected by firefox relay get drawn over-top the in-content popup. I discovered this on the github organization setting page; when deleting an organization, the billing address, gravatar email and public email text boxes all overwrite the "are you sure you want to delete this" popup. This can be worked around by scrolling the page so they don't overlap the textbox.

Steps to reproduce:
1. Create a new organization (such as `test-org-Ahyoo7ba`)
2. Go the the organization settings page (e.g. https://github.com/organizations/test-org-Ahyoo7ba/settings/profile)
3. Click the "delete organization button"
4. Observer that the three relay detected text boxes (i.e. the billing address, gravatar email and public email text boxes) all display on top of the confirmation popup, and without the background shading that the rest of the page gets.

### PR DESCRIPTION
**Pull request by codemist**
…ontent and show the email placeholder

@maxxcrawford  (Is there a limit to how long a commit message can be?)

### CODE REVIEW

**Comment by groovecoder** [comment]
No limit on commit messages. Lots of tools truncate commit messages around 70 characters, so it's best if you can fit a summary on a single line of <70 characters, leave a blank line, and then put more details in the commit message after that. E.g.,

```
fix #10 - Remove z-index to stop textbox drawing over other content

By removing the z-index value from our element, we stop our empty 
textbox from being rendered on top of the underlying email input 
element's placeholder text.
```

**Comment by groovecoder** [comment]
@lesleyjanenorton - do you remember how we settled on the super-high z-index value for our element? Were there other bugs caused by having no z-index value?

**Comment by codemist** [comment]
Removing the z-index property from the textbox wrapper seem to have fixed the drawing over issue as well as restored the placeholder value in the text fields (an issue related to #18 and #13 ) 

**Tested on:** 
reddit.com
academy.newscientist.com/users/sign_in (one of the examples in the comments)
https://www.theverge.com/2020/3/24/21192048/firefox-better-web-scroll-subscription-ad-tracker-blocker-publisher-payment
dropbox.com

**Result:**
The overlay issue However, now the placeholder just stays there on some sites (like Reddit).
image

For Dropbox however, this doesn't seem to be an issue.

File: src/css/relay.css [hunk 1]
```diff
@@ -348,7 +348,6 @@ button.fx-relay-menu-generate-alias-btn[disabled] {
   pointer-events: none;
   padding: 0 !important;
   width: 100%;
-  z-index: 99999999999999999;
```
Comment (maxxcrawford) [inline src/css/relay.css:351]: Is it good to have some `z-index` here? (as is a `1` or a `0`)?

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
I wanted to circle back on this open PR — could you test against some of the sites listed in [the referenced issue](https://github.com/mozilla/fx-private-relay-add-on/issues/10)? 

Additionally, see my previous comment about having some (much smaller) `z-index` value still in place: https://github.com/mozilla/fx-private-relay-add-on/pull/99#pullrequestreview-736498399

**Pull request closed without merging**